src/main/java/baron/commands/CommandManager.java

```diff
@@ -62,6 +62,8 @@ public Command parseCommand(String fullCommand) {
                 return new AddTaskCommand(this.taskManager, this.storage, TaskType.EVENT, commandArg);
             case "delete":
                 return new DeleteTaskCommand(this.taskManager, this.storage, commandArg);
+            case "find":
+                return new FindCommand(this.taskManager, commandArg);
             default:
```

Comment:
In Intellij they allow enhanced switch-case statements, you might want to change your switch-case control flow for better readability. An example of enhanced switch-case statements:

```java
switch (itemCode) {
    case 001 -> System.out.println("It's a laptop!");
    case 002 -> System.out.println("It's a desktop!");
    case 003,004 -> System.out.println("It's a mobile phone!");
}
```

Reply:
Hi De Shao, I didn't know there's such a syntax in Java, looks similar to lambdas. Good to know! And I will definitely use in the future. However, I won't be applying to my code, because I am not sure it is compliant to the coding standard and the Java version (JDK 11, it's a JDK 12-13 feature, I think) of this course. Thank you for the suggestion though.